hockey/indivPlayerFunctions.php

```diff
@@ -0,0 +1,63 @@
+<?php
+function playerInfo($LastName){
+    $info = dbQuery("
+    SELECT *
+    FROM hockey_1617
+    WHERE LastName= :var1",
+    array(
+        "var1"=>$LastName)
+    );
+    return $info->fetch();
+};
+
+function displayPlayerInfo ($name){
+    if($_REQUEST['name']==$name){
+        $result = playerInfo($name);
+        return $result;
+    }
+}
+function playerInfo1516($LastName){
+    $info = dbQuery("
+    SELECT *
+    FROM hockey_1516
+    WHERE LastName= :var1",
+    array(
+        "var1"=>$LastName)
+    );
+    return $info->fetch();
+};
+function displayPlayerInfo1516 ($name){
```

Comment:
Oh, I think I now see that these are two different years of stats? If so, there are a couple of ways to simplify this. What about putting all the data in one single table, but having a "year" column in there?